Model/Logger.php

```diff
@@ -119,29 +132,32 @@ public function force($isForced = true)
      * Get the temp log filename
      *
      * @return string
+     * @throws \Magento\Framework\Exception\FileSystemException
      */
-    public function getPath()
+    public function getPath(): string
     {
-        return $this->directoryList->getPath(DirectoryList::LOG) . DIRECTORY_SEPARATOR . 'taxjar' . DIRECTORY_SEPARATOR . $this->filename;
+        return sprintf(
+            '%s/taxjar/%s',
+            $this->directoryList->getPath(\Magento\Framework\App\Filesystem\DirectoryList::LOG),
+            $this->filename
+        );
     }
 
     /**
      * Save a message to taxjar.log
      *
      * @param string $message
      * @param string $label
-     * @throws LocalizedException
+     * @throws \Magento\Framework\Exception\LocalizedException
      * @return void
      */
-    public function log($message, $label = '')
+    public function log(string $message, string $label = ''): void
```

Comment:
Changing the signature of this method appears to have broken some other portions of the integration. I am getting this error:

Fatal error: Uncaught TypeError: Argument 2 passed to Taxjar\SalesTax\Model\Logger::log() must be of the type string, null given, called in /var/www/html/app/code/Taxjar/SalesTax/Model/Transaction/Refund.php on line 150 and defined in /var/www/html/app/code/Taxjar/SalesTax/Model/Logger.php:154
Stack trace:
#0 /var/www/html/app/code/Taxjar/SalesTax/Model/Transaction/Refund.php(150): Taxjar\SalesTax\Model\Logger->log('Pushing refund ...', NULL)
#1 /var/www/html/app/code/Taxjar/SalesTax/Model/Transaction/Backfill.php(125): Taxjar\SalesTax\Model\Transaction\Refund->push(true)
#2 [internal function]: Taxjar\SalesTax\Model\Transaction\Backfill->process(Object(Magento\AsynchronousOperations\Model\Operation))
#3 /var/www/html/vendor/magento/framework-message-queue/Consumer.php(150): call_user_func(Array, Object(Magento\AsynchronousOperations\Model\Operation))
#4 /var/www/html/vendor/magento/framework-message-queue/Consumer.php(218): Magento\Framework\MessageQueue\Consumer->dispatchMessage(Object(Magento\Framework\MessageQueue\Env in /var/www/html/app/code/Taxjar/SalesTax/Model/Logger.php on line 154

This was when attempting a force backfill that found a single order with a single credit memo. It looks like null is being pass as the second parameter of the log command.